src/components/PendingBusinessTable/PendingBusinessTable.jsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useState } from 'react';
+import { useBackend } from '../../contexts/BackendContext';
+import { Button, Table, Thead, Tbody, Tr, Td } from '@chakra-ui/react';
+import propTypes from 'prop-types';
+
+const PendingBusinessTable = ({ goToBusinessForm }) => {
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+  const TABLE_HEADERS = [
+    'Id',
+    'Type',
+    'Name',
+    'Street',
+    'Zipcode',
+    'State',
+    'Qb Vendor Name',
+    'Qb City State Zip',
+    'Primary Phone',
+    'Backup Phone',
+    'Primary Email',
+    'Comments',
+    'Fax phone',
+    'Contact name',
+    'Website',
+    'Business hours',
+    'Find out',
+    'Onboarding status',
+    'Join date',
+    'Input type status',
+    'Vendor type',
+    'Status',
+    'Pets of the homeless discount',
+    'Updated by',
+    'Updated date time',
+    'Sync to qb',
+    'Veterinary',
+    'Ressoure',
+    'Food',
+    'Donation',
+    'Family shelter',
+    'Wellness',
+    'Spray neuter',
+    'Financial',
+    'Re home',
+    'Er boarding',
+    'Senior',
+    'Cancer',
+    'Dog',
+    'Cat',
+    'Fph phone',
+    'Contact phone',
+    'Web notes',
+    'Internal notes',
+    'Published',
+    'Shelter',
+    'Domestic Violence',
+    'Web Date Init',
+    'Ent Qb',
+    'Service Request',
+    'Inactive',
+    'Final Check',
+    'Created By',
+    'Created Date',
+    'City',
+    'Application',
+  ];
```

Comment:
in general, we're creating a constants.js file and it'll have a table_headers constant variable which would be imported sometime later, don't need to edit this though.